internal/report/report.go

```diff
@@ -433,7 +433,19 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 	}
 
 	if len(syms) == 0 {
-		return fmt.Errorf("no matches found for regexp: %s", o.Symbol)
+		if address == nil { // User provides a symbol instead of an address
+			if len(symbols) == 0 {
+				return fmt.Errorf("no matches found for regexp %s in binaries", o.Symbol)
+			} else {
```

Comment:
nit: remove else and unindent its body. Same below.

Reply:
Done.